spec/game_runner_spec.rb

```diff
@@ -16,11 +16,29 @@
       GRID
 
       stdout = StringIO.new
-      game_start = GameRunner.new(stdout)
+      stdin = StringIO.new
+      game_start = GameRunner.new(stdout, stdin)
 
       game_start.run
 
-      expect(stdout.string).to eq(grid_template)
+      expect(stdout.string).to include(grid_template)
+    end
+
+    context "after rendering the grid, the user is prompted for input" do
+      it "it quits when a user enters a good coordinate" do
+        stdout = StringIO.new
+        stdin = StringIO.new('A2')
+        game_start = GameRunner.new(stdout, stdin)
+        grid_coordinates = ["A1", "A2", "A3", "B1", "B2", "B3", "C1", "C2", "C3"]
```

Comment:
Grid coordinates seems like a private detail. The reason we don't test private details, like how we are thinking of our data is so that we can change our minds when we get a better idea about how to implement the code/data. So, this and the related tests shouldn't be here.

What behavior does the `grid_coordinates` get you that you can test?